common/dynamicconfig/constants.go

```diff
@@ -70,6 +70,11 @@ var (
 		false,
 		`EnableReadFromSecondaryVisibility is the config to enable read from secondary visibility`,
 	)
+	VisibilityEnableShadowReadMode = NewGlobalBoolSetting(
+		"system.VisibilityEnableShadowReadMode",
```

Comment:
First word starts lower case.
```suggestion
		"system.visibilityEnableShadowReadMode",
```